pom.xml

```diff
@@ -467,7 +467,7 @@
           <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>7.8</version>
+            <version>7.9-SNAPSHOT</version>
```

Comment:
With version 7.8, this fix won't affect the maven checkstyle task. CI would still fail. So I change it to 7.9-SNAPSHOT. Is it OK? Why we used stable version but not current SNAPSHOT version before?